swagger.json

```diff
@@ -640,6 +640,175 @@
+        "description": "The Devicestatus endpoint returns information about the Nightscout devicestatus records.",
+        "parameters": [
+          {
+            "name": "find",
+            "in": "query",
+            "description": "The query used to find entries, support nested query syntax, for example `find[dateString][$gte]=2015-08-27`.  All find parameters are interpreted as strings.",
+            "required": false,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "count",
+            "in": "query",
+            "description": "Number of devicestatus records to return.",
+            "required": false,
+            "schema": {
+              "type": "number"
+            }
+          }
+        ],
+        "tags": [
+          "Devicestatus"
+        ],
+        "responses": {
+          "200": {
+            "description": "An array of devicestatus entries",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Devicestatuses"
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Devicestatus"
+        ],
+        "summary": "Add new devicestatus records.",
+        "description": "",
+        "operationId": "addDevicestatuses",
+        "responses": {
+          "200": {
+            "description": "Rejected list of device statuses.  Empty list is success."
+          },
+          "405": {
+            "description": "Invalid input"
+          }
+        },
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Devicestatuses"
+              }
+            }
+          },
+          "description": "Device statuses to be uploaded.",
+          "required": true
+        }
+      },
+      "delete": {
+        "summary": "Delete all Devicestatus records matching query",
+        "description": "The Devicestatus endpoint returns information about the\nNightscout devicestatus records.\n",
+        "parameters": [
+          {
+            "name": "find",
+            "in": "query",
+            "description": "The query used to find entries, support nested query syntax, for\nexample `find[created_at][$gte]=2015-08-27`.  All find parameters\nare interpreted as strings.\n",
+            "required": false,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "count",
+            "in": "query",
+            "description": "Number of records to return.",
```

Comment:
i think this should be `Number of records to delete`? And shouldn't we make `required=true`?

Reply:
Same as above, there is no way to limit the number of records being deleted.